Swift/macroses/common.utils.twig

```diff
@@ -34,21 +38,38 @@
     {%- if field.type.type.baseTypeName == "DateTime" -%}
         {{- self.formatEncodingDate(field) -}}
     {%- elseif field.type.type.baseTypeName == "Decimal" -%}
-        {%- if strong -%}
-            {{ field.name }}
-        {%- else -%}
-            {{ self.formatNullableOrOptional(field.name, field.nullable, field.optional) -}}
-        {%- endif -%}
-        .decimalValue
+        {{ self.formatNullableOrOptional(field.name, strong) -}}.decimalValue
+    {%- elseif field.type.type.baseTypeName == "DateTimeTimestamp" -%}
+        Int({{ self.formatStrongOrNullableOptional(field, strong) -}}.timeIntervalSince1970)
+    {%- elseif field.type.type.baseTypeName == "Color" -%}
+        {{ self.formatStrongOrNullableOptional(field, strong) -}}.hexString
+    {%- elseif field.type.type.baseTypeName == "Url" -%}
+        {{ self.formatStrongOrNullableOptional(field, strong) -}}.absoluteString
+    {%- elseif field.type.type.baseTypeName == "StringDecimal" -%}
+        {{- self.formatEncodingStringDecimal(field) -}}
     {%- else -%}
         {{ field.name }}
     {%- endif -%}
 {% endmacro %}
 
+{% macro formatStrongOrNullableOptional(field, strong) %}
```

Comment:
I don't understand what does mean `strong` here. Can you rename it to something more clear?